cmd/antrea-agent/agent.go

```diff
@@ -238,9 +238,17 @@ func run(o *Options) error {
 	}
 
 	var egressController *egress.EgressController
+	var nodeIP net.IP
+	if nodeConfig.NodeIPv4Addr != nil {
+		nodeIP = nodeConfig.NodeIPv4Addr.IP
+	} else if nodeConfig.NodeIPv6Addr != nil {
+		nodeIP = nodeConfig.NodeIPv6Addr.IP
+	} else {
+		return fmt.Errorf("NodeIPAddr in Node config invalid: %v", nodeConfig)
```

Comment:
`NodeIPAddr in Node config is invalid` or `invalid NodeIPAddr in Node config`

Reply:
Done